src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxListTests.cs

```diff
@@ -305,5 +307,90 @@ public void WithLotsOfChildrenTest()
                 }
             }
         }
+
+        [Theory]
+        [CombinatorialData]
+        public void EnumerateWithManyChildren_Forward(bool trailingSeparator)
```

Comment:
>EnumerateWithManyChildren_Forward [](http://example.com/codeflow?start=20&length=33)

What would be the failure for these tests without the fix? Timeout? If so, consider adding a comment about that.